src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorUnlockVerify.ts

```diff
@@ -38,25 +49,133 @@ export const transformSelectAuthenticatorUnlockVerify: IdxStepTransformer = ({
     stepName,
   );
 
-  const title: TitleElement = {
+  const identifierContainer: IdentifierContainerElement = {
+    type: 'IdentifierContainer',
+    options: {
+      identifier: '',
+    },
+  };
+
+  const unlockAccountTitle: TitleElement = {
     type: 'Title',
     options: {
       content: loc('unlockaccount', 'login'),
     },
   };
 
+  const verifyTitle: TitleElement = {
+    type: 'Title',
+    options: {
+      content: loc('oie.select.authenticators.verify.title', 'login'),
+    },
+  };
+
+  const verifySubtitle: DescriptionElement = {
+    type: 'Description',
+    contentType: 'subtitle',
+    options: {
+      content: loc('oie.select.authenticators.verify.subtitle', 'login'),
+    },
+  };
+
   const identifier = getUIElementWithName('identifier', uischema.elements) as FieldElement;
 
-  const authenticatorListElement: AuthenticatorButtonListElement = {
+  if (identifier) {
+    // add username/identifier from config if provided
+    if (username) {
+      data.identifier = username;
+    } else if (typeof identifier.options.inputMeta.value === 'string') {
+      data.identifier = identifier.options.inputMeta.value;
+    // TODO: OKTA-508744 - to use rememberMe in features once Default values are added widgetProps.
+    // (i.e. rememberMe is default = true in v2)
+    } else if (typeof features?.rememberMe === 'undefined' || features?.rememberMe === true) {
+      const usernameCookie = getUsernameCookie();
+      data.identifier = usernameCookie;
+    }
+  }
```

Comment:
I brought it over from the primary auth form because I thought it would be good UX and we were trying to match the sign-in flow as closely as possible, but I'm not sure if it's necessary. I'll ping Alex

Reply:
Alex said we could support pre-population so I'm marking this as resolved